Modules/Filtering/ImageGrid/include/itkOrientImageFilter.h

```diff
@@ -70,72 +67,6 @@ namespace itk
  * for the itk::Image object and the Image.Direction direction cosine
  * matrix created from the file.
  *
- * As an example, if you wished to keep all images within your program in the
- * orientation corresponding to the Analyze file format's 'CORONAL' orientation
- * you could do something like the following
- *
-   \code
-   // DEPRECATED -- using metadata for orientation is no longer supported
-   //
-   #include "itkAnalyzeImageIO.h"
-   #include "itkMetaDataObject.h"
-   #include "itkImage.h"
-   #include "itkOrientImageFilter.h"
-   using ImageType = itk::Image<unsigned char,3>;
-   using ImageReaderType = itk::ImageFileReader< TestImageType >;
-   ImageType::Pointer ReadAnalyzeFile(const char *path)
-   {
-     itk::AnalyzeImageIO::Pointer io = itk::AnalyzeImageIO::New();
-     auto fileReader = ImageReaderType::New();
-     fileReader->SetImageIO(io);
-     fileReader->SetFileName(path);
-     fileReader->Update();
-     ImageType::Pointer rval = fileReader->GetOutput();
-
-   // DEPRECATED -- use direction cosines
-   //
-    itk::SpatialOrientation::ValidCoordinateOrientationFlags fileOrientation;
-    itk::ExposeMetaData<itk::SpatialOrientation::ValidCoordinateOrientationFlags>
-      (rval->GetMetaDataDictionary(),itk::ITK_CoordinateOrientation,fileOrientation);
-     itk::OrientImageFilter<ImageType,ImageType>::Pointer orienter =
-       itk::OrientImageFilter<ImageType,ImageType>::New();
-     orienter->SetGivenCoordinateOrientation(fileOrientation); // deprecated
-
-     orienter->SetDesiredCoordinateOrientation(itk::SpatialOrientation::ITK_COORDINATE_ORIENTATION_RIP);
-     orienter->SetInput(rval);
-     orienter->Update();
-     rval = orienter->GetOutput();
-     return rval;
-   }
-   \endcode
- *
- * Or, using the direction cosines of the image,
-   \code
-   #include "itkAnalyzeImageIO.h"
-   #include "itkImage.h"
-   #include "itkOrientImageFilter.h"
-   using ImageType = itk::Image<unsigned char,3>;
-   using ImageReaderType = itk::ImageFileReader< ImageType >;
-   ImageType::Pointer ReadAnalyzeFile(const char *path)
-   {
-     itk::AnalyzeImageIO::Pointer io = itk::AnalyzeImageIO::New();
-     auto fileReader = ImageReaderType::New();
```

Comment:
It would be good to retain an example here. We should update this one, and simplify it to use `itk::ReadImage<>()`.

Reply:
I am not a big fan of having a large amount of untested example code in the documentation as it can easily become incorrect.

Reply:
We should create an example for it, so it appears similarly to [ResampleImageFilter](https://itk.org/Doxygen/html/classitk_1_1ResampleImageFilter.html). I assume just adding an example [here](https://github.com/InsightSoftwareConsortium/ITK/tree/master/Examples/Filtering) which uses `Orient` filter should do the trick.